openpathsampling/tests/test_helpers.py

```diff
@@ -232,4 +239,41 @@ def current_snapshot(self, snapshot):
 
     def generate_next_frame(self):
         self._current_snapshot = None
-        return self.current_snapshot
\ No newline at end of file
+        return self.current_snapshot
+
+
+class IdentityPathMover(paths.PathMover):
```

Comment:
Yes, I had it first in PathMover and then I moved it since it was only part of the test...